x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/TransactionTabs.tsx

```diff
@@ -7,38 +7,38 @@
 import { EuiSpacer, EuiTab, EuiTabs } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { Location } from 'history';
-import { get } from 'lodash';
 import React from 'react';
-import styled from 'styled-components';
 import { Transaction } from '../../../../../typings/es_schemas/ui/Transaction';
 import { IUrlParams } from '../../../../store/urlParams';
-import { px, units } from '../../../../style/variables';
-import { HeightRetainer } from '../../../shared/HeightRetainer';
 import { fromQuery, history, toQuery } from '../../../shared/Links/url_helpers';
 import { PropertiesTable } from '../../../shared/PropertiesTable';
-import {
-  getCurrentTab,
-  getTabsFromObject
-} from '../../../shared/PropertiesTable/tabConfig';
 import { WaterfallContainer } from './WaterfallContainer';
 import { IWaterfall } from './WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers';
 
-const TableContainer = styled.div`
-  padding: ${px(units.plus)} ${px(units.plus)} 0;
-`;
-
 interface TimelineTab {
   key: 'timeline';
   label: string;
 }
 
```

Comment:
is there any benefit for differentiating the TimelineTab and MetadataTab types? Can we combine them into a single interface `ViewTab` or something like that?

Reply:
Actually now that you mention it, there's no reason to not just let TS infer the types of these altogether. Thanks!